etc/skydive.yml.default

```diff
@@ -29,6 +29,7 @@ analyzer:
   # Must be different than the agent
   # X509_cert: /etc/ssl/certs/analyzer.domain.com.crt
   # X509_key:  /etc/ssl/certs/analyzer.domain.com.key
+  # topology_filter: "{\"TOR1\":\"Name : TOR1\",\"AGGR1\":\"Name : AGGR1\",\"ALL\":\".*\"}"
```

Comment:
That's a shame that config API only returns simple strings.

I'm working on a patch so that you can retrieve hash directly from the API, so that you don't have to parse the string on the JS side

Reply:
OK, I will make a change that will use the new API once it will be ready.
Thanks!
Lera